code/_onclick/ventcrawl.dm

```diff
@@ -181,6 +181,18 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 							if(!ventcrawl_carry())
 								return
 
+							if(stat)
+								to_chat(src, "<span class='danger'>You must be conscious to do this!</span>")
+								return
+
+							if(locked_to)
+								to_chat(src, "<span class='warning'>You can't vent crawl while you're restrained!</span>")
+								return
+
+							if(lying)
```

Comment:
lying doesn't always means you're stunned.

Reply:
I'm using the same checks that you put in #15723, should I change them too?

Reply:
:thinking: I mean i just changed their order

Reply:
But maybe?